sdk/inc/azure/core/az_result.h

```diff
@@ -156,8 +156,8 @@ AZ_NODISCARD AZ_INLINE bool az_result_failed(az_result result)
  *
  * @param[in] result Result value to check for success.
  *
- * @retval `true` The operation that returned this \p result was successful.
- * @retval `false` The operation that returned this \p result failed.
+ * @retval true The operation that returned this \p result was successful.
+ * @retval false The operation that returned this \p result failed.
```

Comment:
see: https://github.com/Azure/azure-sdk-for-c/pull/1344 referring to this same code

Reply:
Lol, yep, @antkmsft, feel free to close this one as it's already done.